### PR TITLE
event participants & event list fix

### PR DESCRIPTION
**Pull request by sandorbanvitagroup**

### CODE REVIEW

**Review by danielrus — DISMISSED** [review]

File: src/app/events/event-components/event-participants/event-participants.component.ts [hunk 1]
```diff
@@ -14,10 +15,14 @@ export class EventParticipantsComponent implements OnInit {
...
-
-  constructor(public eventParticipantService: EventParticipantService) {}
+  presetFilters: Filter[] = [];
+  constructor(public eventParticipantService: EventParticipantService, private route: Router) {}
 
   ngOnInit(): void {
+    const urlParts = this.route.url.split('/');
+    const eventId = urlParts[3];
+
+    this.presetFilters = [{ field: 'event', value: { uuid: eventId } }];
```
Comment (adriandumavitagroup) [inline src/app/events/event-components/event-participants/event-participants.component.ts]: this is not ok. It needs to have a subscribe for the url. And also, do not use url.split. Use the this.activatedRoute.params or something else

Comment (sandorbanvitagroup) [reply src/app/events/event-components/event-participants/event-participants.component.ts]: Resolved.

**Review by adriandumavitagroup — DISMISSED** [review]

File: src/app/events/event-components/event-participants/event-participants.component.ts [hunk 2]
```diff
@@ -14,10 +15,16 @@ export class EventParticipantsComponent implements OnInit {
...
-  constructor(public eventParticipantService: EventParticipantService) {}
+  presetFilters: Filter[] = [];
+  constructor(
+    public eventParticipantService: EventParticipantService,
+    private activatedRoute: ActivatedRoute
+  ) {}
 
   ngOnInit(): void {
     this.defaultColumns = defaultColumnDefs;
+    this.activatedRoute.params.subscribe((params) => {
```
Comment (adriandumavitagroup) [inline src/app/events/event-components/event-participants/event-participants.component.ts]: unsubscribe

Comment (sandorbanvitagroup) [reply src/app/events/event-components/event-participants/event-participants.component.ts]: Done.

**Review by adriandumavitagroup — APPROVED** [review]

**Review by danielrus — APPROVED** [review]

**Pull request merged**